tests/network/vmi_multus.go

```diff
@@ -121,15 +147,10 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 
 		configureNodeNetwork(virtClient)
 
-		result := virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "linux-bridge-net-vlan100")).
-			Body([]byte(fmt.Sprintf(linuxBridgeConfCRD, "linux-bridge-net-vlan100", tests.NamespaceTestDefault))).
-			Do(context.Background())
-		Expect(result.Error()).NotTo(HaveOccurred())
+		Expect(createBridgeNetworkAttachementDefinition("linux-bridge-net-vlan100", "100", ""))
```

Comment:
Ok, you `Expect(...)` this to what ? You're missing the matcher part - i.e.`To(Succeed())` .